single-campaign.php

```diff
@@ -100,15 +100,13 @@ static function ( $page ) {
 $context['custom_styles']   = $custom_styles;
 $context['css_vars']        = PostCampaign::css_vars( $campaign_meta );
 
-if ( $theme_name && 'default' !== $theme_name ) {
-	$context['custom_font_families'] = [
-		'Montserrat:300,400,500,600,700,800',
-		'Kanit:400,500,600,800',
-		'Open+Sans:400,500,600,800',
-		'Anton:400,500,600,800',
-		'Amiko:400,500,600,800',
-	];
-}
+$context['custom_font_families'] = [
```

Comment:
I removed the check because some of these can be used for the default theme as well.